src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/Vault/SetAzureRmRecoveryServicesAsrVaultSettings.cs

```diff
@@ -75,25 +76,28 @@ public override void ExecuteSiteRecoveryCmdlet()
             {
                 using (var powerShell = System.Management.Automation.PowerShell.Create())
```

Comment:
@viverm if the `powerShell` object is not being used anymore, we should remove the using